compute/disks/create_empty_disk.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+// [START compute_disk_create_empty_disk]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	compute "cloud.google.com/go/compute/apiv1"
+	computepb "google.golang.org/genproto/googleapis/cloud/compute/v1"
+	"google.golang.org/protobuf/proto"
+)
+
+// createEmptyDisk creates a new empty disk in a project in given zone.
+func createEmptyDisk(
+	w io.Writer,
+	projectID, zone, diskName, diskType string,
```

Comment:
Why is `SizeGb` omitted from these params?

Reply:
We decided previously to hard code it: https://github.com/GoogleCloudPlatform/golang-samples/pull/2623#discussion_r915064713